parsers/jscript.c

```diff
@@ -1199,6 +1199,13 @@ static bool parseMethods (tokenInfo *const token, tokenInfo *const class)
 	 *	   validMethod    : function(a,b) {}
 	 *	   'validMethod2' : function(a,b) {}
      *     container.dirtyTab = {'url': false, 'title':false, 'snapshot':false, '*': false}
+     *
+     * ES6 methods:
+     *     property(...) {}
+     *     *generator() {}
+     * FIXME: what to do with computed names?
+     *     [property]() {}
+     *     *[generator]() {}
```

Comment:
You can write about these hard (?) question/decision in designing the parser at docs/parser-jscript.rst (or docs/parser-javascript.rst).

This is just a comment, not a requirement.